app/src/components/new-request/search-components/business-lookup-fetch.vue

```diff
@@ -118,6 +118,8 @@ export default class BusinessLookupFetch extends Mixins(CommonMixin, SearchMixin
       if (this.getSearchBusiness) {
         // set the from business for alteration (conversion)
         this.setOriginEntityTypeCd(this.getSearchBusiness.legalType)
+        // check if business is in Lear and set store value of isLearBusiness flag
+        this.checkBusinessInLear(this.getSearchBusiness?.identifier)
```

Comment:
You don't need conditional chaining because the object is already validated on line 118.

Conditional chaining is a double-edged sword as it can mask errors. Not here, but generally we need to be careful with it, and here you don't need it.

Reply:
Yes! Removing